bcda/models/models.go

```diff
@@ -82,11 +84,11 @@ func GetATOPrivateKey() *rsa.PrivateKey {
 	return secutils.OpenPrivateKeyFile(atoPrivateKeyFile)
 }
 
-func CreateACO(name string) (uuid.UUID, error) {
+func CreateACO(name string, cmsID *string) (uuid.UUID, error) {
```

Comment:
why a pointer here?  it works fine but just wondering for my own knowledge

Reply:
To allow it to be null for adding to the db. We could also assume empty string means nil and not use a pointer, but this seemed a little simpler.